text/0003-encrypted-memos.md

```diff
@@ -3,6 +3,8 @@
 - RFC PR: [mobilecoinfoundation/mcips#3](https://github.com/mobilecoinfoundation/mcips/pull/3)
 - MobileCoin Epic: None
 
+This MCIP describes a memo which has had its size adjusted from 46 to 66 in MCIP #24 
```

Comment:
Consider linking to the MCIP

```suggestion
This MCIP describes a memo which has had its size adjusted from 46 to 66 in [MCIP #24](0024-increase-memo-field-size.md)
```

Reply:
@jcape I'm actually not sure why you asked me to remove the link. Do you mind clarifying this?

Reply:
This appears unnecessary since the #24 seems to automatically link it.